docs/ref/models/querysets.txt

```diff
@@ -2341,8 +2343,22 @@ performance and a significant reduction in memory.
 Note that using ``iterator()`` on a ``QuerySet`` which has already been
 evaluated will force it to evaluate again, repeating the query.
 
-Also, use of ``iterator()`` causes previous ``prefetch_related()`` calls to be
-ignored since these two optimizations do not make sense together.
+.. versionchanged:: 4.1
+
+    ``iterator()`` is compatible with previous calls to ``prefetch_related()``
+    as long as ``chunk_size`` is given. Larger values will necessitate fewer
+    queries to accomplish the prefetching at the cost of greater memory usage.
+
+    As the maximum number of terms in an SQL ``IN`` clause on SQLite is 999,
```

Comment:
Perhaps this text is fine as-is, given it is talking about the lowest common denominator supported by Django, but AFAIK recent versions of SQLite (`3.32+`) aren't encumbered by this, so _one_ day when that's the minimum supported version, it won't be relevant (although maybe that day is 'today' if the `DatabaseFeatures` were modified?)

Reply:
I would leave `999` because `SQLite` has other limits that can cause a crash, e.g. `SQLITE_MAX_COLUMN` (see f91b5a7e4b2ac436b1fe3065b23217543d0f54c1 and ticket-26063), or `SQLITE_MAX_LENGTH`.

Reply:
Also, the maximum number of terms is also limited on Oracle by `2^16 - 1`. Maybe we could use a more general note, e.g.
```
    On some databases (e.g. Oracle,
    `SQLite <https://www.sqlite.org/limits.html#max_variable_number>`_), the
    maximum number of terms in an SQL ``IN`` clause might be limited. Hence
    values below this limit should be used. (In particular, when prefetching
    across two or more relations, a ``chunk_size`` should be small enough that
    the anticipated number of results for each prefetched relation still falls
    below the limit.)
```